packages/gatsby-theme-patternfly-org/components/footer/footer.js

```diff
@@ -156,11 +155,7 @@ export class Footer extends React.Component {
AuMzI0OCBMODkuNjk1ODkwNCwwLjMyNDggTDg5LjY5NTg5MDQsOS4zNDcyIEw4OC4xNTUyNTExLDkuMzQ4OCBMODguMTU1MjUxMSw5LjM0ODggWiIgaWQ9IlBhdGgiPjwvcGF0aD4KICAgICAgICAgICAgICAgICAgICAgICAgPHBvbHlnb24gaWQ9IlBhdGgiIHBvaW50cz0iOTcuNDU5MDE4MyA5LjM0ODggOTcuNDU5MDE4MyAwLjMyNjQgMTAzLjUwMTcxMiAwLjMyNjQgMTAzLjUwMTcxMiAxLjk4MTYgOTkuMjAwMjI4MyAxLjk4MTYgOTkuMjAwMjI4MyAzLjg3NTIgMTAxLjkyNjcxMiAzLjg3NTIgMTAxLjkyNjcxMiA1LjUyMzIgOTkuMjAwMjI4MyA1LjUyMzIgOTkuMjAwMjI4MyA5LjM0OTYiPjwvcG9seWdvbj4KICAgICAgICAgICAgICAgICAgICAgICAgPHBvbHlnb24gaWQ9IlBhdGgiIHBvaW50cz0iMTEwLjU1NTI1MSA5LjM0ODggMTEwLjU1NTI1MSAwLjMyNjQgMTEyLjI5ODA1OSAwLjMyNjQgMTEyLjI5ODA1OSA3LjY3NTIgMTE2LjY0NjY4OSA3LjY3NTIgMTE2LjY0NjY4OSA5LjM0ODgiPjwvcG9seWdvbj4KICAgICAgICAgICAgICAgICAgICAgICAgPHBvbHlnb24gaWQ9IlBhdGgiIHBvaW50cz0iMTI0Ljk3Mzk3MyA5LjM0ODggMTI0Ljk3Mzk3MyA1LjgzMiAxMjEuNjM0NTg5IDAuMzI2NCAxMjMuNjA4MzMzIDAuMzI2NCAxMjUuODM1Mzg4IDQuMDE0NCAxMjguMDU4NDQ3IDAuMzI2NCAxMzAuMDMzNzkgMC4zMjY0IDEyNi42OTUyMDUgNS44MzIgMTI2LjY5NTIwNSA5LjM0ODgiPjwvcG9seWdvbj4KICAgICAgICAgICAgICAgICAgICA8L2c+CiAgICAgICAgICAgICAgICAgICAgPHBhdGggZD0iTTMzLjUxNDQ5NzcsMTYuNTYwOCBMMTYuNzU4NDQ3NSwwLjE1MzYgTDAsMTYuNTYwOCBMNi42ODAzNjUzLDI2LjAzNiBMMTEuNzc2OTQwNiwyNC44NjQgTDE2Ljc5MDQxMSwzMS45MjA4IEwyMS43OTQyOTIyLDI0Ljg3NjggTDI2LjgzNDEzMjQsMjYuMDM1MiBMMzMuNTE0NDk3NywxNi41NjA4IFogTTE2LjA2MjQ0MjksMy4xOTYgTDEzLjg5MjEyMzMsMjMuMjc2OCBMOS43NDgwNTkzNiwxNy42MTc2IEwxNi4wNjI0NDI5LDMuMTk2IFogTTkuMjkzMzc5LDE2Ljk5NjggTDYuNzEyMzI4NzcsMTMuNDcyIEwxNS41ODc3ODU0LDIuNjggTDkuMjkzMzc5LDE2Ljk5NjggWiBNMTMuMjA0MTA5NiwyMy40NjQgTDEzLjIwNDEwOTYsMjMuNDY0IEwxMy40NDU0MzM4LDIzLjc5NjggTDEyLjA2MTQxNTUsMjQuMTE0NCBMOC44NjY2NjY2NywxOS42MTg0IEw5LjQzNzIxNDYxLDE4LjMyMDggTDEzLjIwNDEwOTYsMjMuNDY0IFogTTE2Ljc1NzY0ODQsMi45Mjk2IEwxOC45MzkxNTUzLDI0LjIxODQgTDE2Ljc1Mjg1MzksMjcuMjE3NiBMMTQuNTc0NTQzNCwyNC4yMTg0IEwxNi43NTc2NDg0LDIuOTI5NiBaIE0yMC4zMDk1ODksMjMuNDY0OCBMMjQuMTE4MDM2NSwxOC4yNjQ4IEwyNC43MTQxNTUzLDE5LjYxODQgTDIxLjUxMDYxNjQsMjQuMTI4IEwyMC4wNzE0NjEyLDIzLjc5NjggTDIwLjMxMTk4NjMsMjMuNDY2NCBMMjAuMzA5NTg5LDIzLjQ2NDggWiBNMTkuNjIyMzc0NCwyMy4yNzY4IEwxNy40ODAwMjI4LDMuMTc5MiBMMjMuODA3OTkwOSwxNy41NjA4IEwxOS42MjIzNzQ0LDIzLjI3NjggWiBNMTcuOTg1MDQ1NywyLjY3NjggTDI2LjgwMjE2ODksMTMuNDcyIEwyNC4yNjI2NzEyLDE2Ljk0IEwxNy45ODUwNDU3LDIuNjc2OCBaIE02Ljk2NjQzODM2LDI1LjI4NTYgTDAuODcwMjA1NDc5LDE2LjY0MDggTDEyLjc5ODk3MjYsNC45NjcyIEw1Ljg3MDg5MDQxLDEzLjQ1MDQgTDguOTg0MTMyNDIsMTcuNzAwOCBMOC4xMDU5MzYwNywxOS42OTYgTDExLjM1OTgxNzQsMjQuMjc2OCBMNi45NjY0MzgzNiwyNS4yODU2IFogTTE2Ljc5MDQxMSwzMC43NzEyIEwxMi40Nzg1Mzg4LDI0LjcwMzIgTDEzLjg3MDU0NzksMjQuMzgyNCBMMTYuNzUxMjU1NywyOC4zNDk2IEwxOS42NDE1NTI1LDI0LjM4MjQgTDIxLjA5MTA5NTksMjQuNzE2OCBMMTYuNzkwNDExLDMwLjc3MTIgWiBNMjIuMjExNDE1NSwyNC4yODk2IEwyNS40NzQwODY4LDE5LjY5NiBMMjQuNTcxOTE3OCwxNy42NDQ4IEwyNy42NDM2MDczLDEzLjQ1MDQgTDIwLjcxNTUyNTEsNC45NjcyIEwzMi42NDM0OTMyLDE2LjY0MDggTDI2LjU0ODA1OTQsMjUuMjg2NCBMMjIuMjExNDE1NSwyNC4yODk2IFoiIGlkPSJTaGFwZSIgZmlsbD0idXJsKCNsaW5lYXJHcmFkaWVudC0xKSI+PC9wYXRoPgogICAgICAgICAgICAgICAgPC9nPgogICAgICAgICAgICA8L2c+CiAgICAgICAgPC9nPgogICAgPC9nPgo8L3N2Zz4=" alt="Patternfly Logo" /></a>
```

Comment:
In trainingLayout.js and sideNavLayout.js, we're using the Brand component, which will eliminate the need for the CSS selectors. We shouldn't need to embed the SVG, but here is an example.

https://github.com/patternfly/patternfly-org/blob/master/packages/gatsby-theme-patternfly-org/layouts/trainingLayout/trainingLayout.js#L40

Reply:
Fixed